### ISSUE TITLE
Designer - Ticket Wallet Design Updates

### ISSUE THREAD

**Issue by Mcscott142** [revelrylabs/bn-mobile-react#158]
## Background
We need to make a few tweaks in order to get some pages up to date with the production mocks. This is a checklist for those updates. 

- [ ] cards should have rounded edges
- [ ] cards should have a black gradient below the text so it’s readable
- [ ] The header should not scroll up (maybe break this one out?)

via @zmelamed 
<img width="1639" alt="ios_app__zach_-_dev_prep__sketch 3" src="https://user-images.githubusercontent.com/6877299/46614286-109a5b00-cadb-11e8-9a25-ebb097a76405.png">

**Comment by Mcscott142** [comment]
Breaking these out 
- [ ] The header should not scroll up (maybe break this one out?)
- [ ] The tabs and header should be updated to match the spec’d version (We can split this out into another ticket unless it's a quick fix. Lower priority.)

**Comment by brittanygay** [comment]
Pulling this in to work on next (sprint tickets are complete on my side, this is not in our current sprint).

**Comment by brittanygay** [comment]
Updates are now approved and merged. Once deployment is complete this will be ready for QA
